ckan/config/declaration/option.py

```diff
@@ -363,12 +364,13 @@ def _validators_from_string(s: str) -> list[Validator]:
     from ckan.logic import get_validator
 
     out = []
-    parts = s.split()
+    parts = re.split(r"\s+", s) if s else s.split()
```

Comment:
Can you explain it? The only difference I see it that string that consists of multiple space characters - `      ` - will be turned into `['', '']` instead of `[]`.

Reply:
i've used the regex becase we can improve how we can split the incoming string but also `s.split` didnt returned the values i was expecting.  And because  the `[' ']` im addint the `if` statement. I was aware that this is not the greatest solution but only workaround to get current implementation to work